### PR TITLE
Style application

### PR DESCRIPTION
**Pull request by Daron976**
### In this PR:
- Added styling to the application

### CODE REVIEW

File: src/components/bookItem.js [hunk 1]
```diff
@@ -39,14 +43,29 @@ const Books = (props) => {
...
+          <p className="status-details">Incomplete</p>
+        </div>
+      </div>
+      <section className="position">
+        <h2 className="pos-header">CURRENT CHAPTER</h2>
+        <p className="pos-details">Chapter 1</p>
+        <div>
+          <button type="button" name="update" id="update">UPDATE PREGRESS</button>
+        </div>
+      </section>
```
Comment (VICTOR-CHUKWU) [inline src/components/bookItem.js]: - [ ] Kindly know that this milestone is to help you improve your styling, currently you have just skipped just few things to do, kindly see the images I pasted below and fix them
your design :point_down: 
WhatsApp Image 2022-11-11 at 1 48 24 PM
there is a border between the status and the button which is currently missing in your design, kindly add that
zeplin design :point_down: 
Screenshot from 2022-11-11 13-54-30

File: src/components/bookItem.js [hunk 2]
```diff
@@ -39,14 +43,29 @@ const Books = (props) => {
...
+      <div className="status">
+        <img src={spinner} alt="progression circle" className="spinner-img" />
+        <div>
+          <p className="percentage">0%</p>
+          <p className="status-details">Incomplete</p>
+        </div>
+      </div>
+      <section className="position">
+        <h2 className="pos-header">CURRENT CHAPTER</h2>
+        <p className="pos-details">Chapter 1</p>
```
Comment (VICTOR-CHUKWU) [inline src/components/bookItem.js:56]: - [ ] also this looks darker in your design than it is in zeplin design, kindly fix this

**Review by VICTOR-CHUKWU — CHANGES_REQUESTED** [review]
Hi @Daron976  👋  

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

almost

## Highlights
- Good PR
- Good commit
- All linters passed


## Required Changes ♻️

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by BertrandConxy — APPROVED** [review]
Hi @Daron976, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights 💪 
- No linter errors
- project requirements are met
- project works fine
- good PR description
- correct flow is used

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @BertrandConxy in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**